### DISCUSSION TITLE
About "server.on" function

### DISCUSSION THREAD

**Discussion by AntonioFromBrazil** [JAndrassy/WiFiEspAT#92] [Ideas]
I wonder if the function server.on would be implemented... This function is present on espressif package libraries and is very usefull

**Comment by JAndrassy** [comment]
https://github.com/JAndrassy/WiFiEspAT/blob/master/examples/Advanced/HelloServer/HelloServer.ino

**Reply by AntonioFromBrazil** [reply]
Wow... great... sorry for that !!!
